packages/dashboard/src/components/robots/robot-page.tsx

```diff
@@ -162,7 +168,7 @@ export function RobotPage() {
       setIngestors(ingestors);
       setFleets(fleets);
     })();
-  }, [rmfIngress]);
+  }, [rmfIngress, _triggerRender, selectedRobot]);
```

Comment:
Adding these dependencies causes all data to be fetched every render, `selectRobot` is probably not needed as well.

Reply:
Yes, that is the whole, idea, because the data are not fetched automatically, and the user need to refresh the page to see any update, in the table or the panel.